packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -332,35 +337,138 @@ export const useTransactionHistory = (
       refreshWhenOffline: false,
       refreshWhenHidden: false,
       revalidateFirstPage: false,
-      refreshInterval: 0
+      keepPreviousData: true,
+      dedupingInterval: 1_000_000
```

Comment:
Arbitrary, we just make sure network request aren't duplicated. So what 1_000_000 (in ms) means is that duplicate requests won't be repeated unless ~16 minutes have passed in between requests. We want this because we don't have to revalidate any requests even if they are duplicate